config/samples/go_example_loadtest.yaml

```diff
@@ -27,6 +27,7 @@ spec:
     language: go
     name: '0'
     run:
+    - name: "worker"
```

Comment:
I don't think the examples should have container names, since htey are replaced by default.

Reply:
Name is a required field in [Container](https://pkg.go.dev/k8s.io/api/core/v1#Container), it will not pass the validation if we don't have a name. Do we want to always change the first run container's name as run?